internal/command/scale/count_machines.go

```diff
@@ -192,12 +286,20 @@ func computeActions(machines []*api.Machine, expectedGroupCounts map[string]int,
 		mConfig.Standbys = nil
 
 		for regionName, delta := range regionDiffs {
+			var availableVolumes []api.Volume
+			if len(mConfig.Mounts) > 0 {
+				availableVolumes = lo.Filter(volumes, func(v api.Volume, _ int) bool {
+					return !v.IsAttached() && v.Region == regionName
+				})
```

Comment:
there is an edge case, not so common but still possible, the mounts can be shared with multiple process groups.
If that were the case, scaling up will reuse the same volume list for different process groups